esp-hal/src/aes/esp32.rs

```diff
@@ -1,4 +1,4 @@
-use crate::aes::{Aes, Aes128, Aes192, Aes256, AesFlavour, Endianness, ALIGN_SIZE};
+use crate::aes::{Aes, Aes128, Aes192, Aes256, AesFlavour, Endianness, Mode, ALIGN_SIZE};
```

Comment:
This is going to interact interestingly with the `Mode` trait when async support is added.

Reply:
There's always the ability to fully qualify something. Regardless, going from an enum to a u8 to a u32 was just nonsense.